tests/integration_tests/test_11001_maus.py

```diff
@@ -34,5 +34,6 @@ def test_maus_creation_11001(self, datafiles):
             actual_deep_ahb, ensure_ascii=True, sort_keys=True, indent=True
         )
         assert actual_maus_json is not None
-        maus_file_path = Path("edifact-templates/maus/FV2110/UTILMD/11001_maus.json")
+        d = Path(__file__).resolve().parents[0]
```

Comment:
was ist "d" und warum braucht man es?

Reply:
können wir das vllt in den integration_test_helper auslagern? unter einem sprechenden namen wie
```python
def get_integration_test_root(file_path:str)->Path:
   return Path(file_path).resolve().parents[0]
```

Reply:
aaah hier ist der test ^^
hatte den gestern nicht mehr gefunden. Kann sein dass man das nicht braucht mit dem fixture.

Reply:
Jop ist nicht mehr notwendig :)
823cf314877ed6bfb6d4e35de4b140363060d93c